.travis.yml

```diff
@@ -1,15 +1,24 @@
 language: python
 python:
   - "3.6"
-# command to install dependencies
 services:
-  - mysql
+  # - mysql
 before_script:
-  - mysql -e 'CREATE DATABASE travisdb;'
+  # - mysql -e 'CREATE DATABASE travisdb;'
 install:
-  - python setup.py -q install
-script:
+  - python setup.py install -q
+script:  
+  - python officemanager/manage.py makemigrations
   - python officemanager/manage.py migrate
```

Comment:
The migrate command is redundant here. Django applies all migrations during the unit tests.